src/cargo/util/command_prelude.rs

```diff
@@ -102,6 +103,13 @@ pub trait CommandExt: Sized {
         )
     }
 
+    fn arg_unsupported_keep_going(self) -> Self {
+        let msg = "use `--no-fail-fast` to run as many tests as possible regardless of failure";
+        let value_parser =
+            UnknownArgumentValueParser::suggest_arg("--no-fail-fast").and_suggest(msg);
```

Comment:
I feel like `suggest_arg` and `suggest` are a bit redundant...

Reply:
You're absolutely right! Fixed.